### PR TITLE
Use route helper with current route watcher

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Active route helpers are not reactive and need to be updated on route change. This PR does that for the AppBar.

## Related Issue
- Fixes https://github.com/owncloud/web/issues/6326

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/22459/69/1
The following scenarios passed on retry:
- webUIUserJourney/journey1.feature:11

File: packages/web-app-files/src/components/AppBar/AppBar.vue [hunk 1]
```diff
@@ -250,6 +261,12 @@ export default {
     },
 
     breadcrumbs() {
+      console.log(
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/AppBar/AppBar.vue:264]: Do you want these merged?

**Review by pascalwengerter — APPROVED** [review]
Can confirm it solves the issue 🥳  CI is green, just a small question

File: packages/web-app-files/src/components/AppBar/AppBar.vue (hunk 1 above)
Comment (kulmann) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:264]: nope 😆 thanks for spotting

Comment (pascalwengerter) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:264]: Remove & force-merge? 😬

Comment (kulmann) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:264]: Nope, I'm trying a different solution locally with a `watchActiveLocation` helper. Not sure if that'll work, yet. But the code change I made in this PR looks too verbose for potentially being applied in other locations.

Comment (kulmann) [reply packages/web-app-files/src/components/AppBar/AppBar.vue]: Please review again 😃

**Pull request merged**